src/Game/Client.java

```diff
@@ -757,6 +758,57 @@ public static void checkForUpdate(boolean announceIfUpToDate) {
 		}
 	}
 	
+	/**
+	 * Index fix after menu swap of redrawMenuHook
+	 * 
+	 * @param menuindex - the index of the menu
+	 * @return the fixed index
+	 */
+	public static int swapUseMenuHook(int menuindex) {
+		int newmenuindex = menuindex;
+		if (newmenuindex == 635) {
+			newmenuindex = 650;
+		} else {
+		}
+		return newmenuindex;
+	}
+	
+	/**
+	 * Redraw right click menu add item hook, only interested from the portion of items with special commands 640 or
+	 * none
+	 * 
+	 * @param instance - the instance of common menu
+	 * @param n - some index sent over
+	 * @param index - the item index
+	 * @param itemCommand - item command
+	 * @param itemName - item name
+	 */
+	public static void redrawMenuHook(Object instance, int n, int index, String itemCommand, String itemName) {
+		
+		if (instance != null) {
+			try {
+				if (!itemCommand.equals("")) {
+					if (!Item.shouldPatch(index)) {
+						Reflection.menuGen.invoke(instance, n, 640, false, itemCommand, "@lre@" + itemName);
+						Reflection.menuGen.invoke(instance, n, 650, false, "Use", "@lre@" + itemName);
```

Comment:
Does the text "Use" matches how it does in-game? I noticed earlier your screenshot had "eat".

Reply:
use functionality is exactly the same, just for the part where its swapping it gets temporarily 635, but then in another hook it gets reincorporated as 650

Reply:
I'm talking about the text you set here, is it "Use" or "use"? If it's "use", make sure it reflects that.

Reply:
its "Use" and was before "Use", that text remains the same

Reply:
Ok, and the same is true for the remaining options?

Reply:
yea they were left as is